cypress/integration/admin.spec.js

```diff
@@ -0,0 +1,44 @@
+const { join } = require('path');
+
+context('Admin interface', () => {
+  it('should redirect anonymous users to the login page', function() {
+    cy.visit('/admin')
+    cy.location().should('match', /\/admin\/login\//)
+  })
+
+  function login() {
+    cy.visit('/admin')
+    cy.readFile(join(__dirname, '..', '..', 'CreeDictionary', '.cypress-user.json')).then(cypressUser => {
+      cy.get('#id_username').type(cypressUser.username)
+      cy.get('#id_password').type(cypressUser.password)
+      cy.get('.submit-row > input').click()
+    })
+    cy.location('pathname').should('eq', '/admin/')
```

Comment:
Should this assertion be inside the callback for `cy.readFile().then()`? I _think_ the way it is currently, this assertion will resolve as true before Cypress finishes logging in. Or does this timing not matter here?